### ISSUE TITLE
Android 9 support

### ISSUE THREAD

**Issue by fmendoza** [taivo/parse-push-plugin#136]
Any advice on how to get this working on Android 9? The main issue is that the deviceToken is saved as undefined in database.

**Comment by NicksonYap** [comment]
Yes, this is also the case for me, took a lot to realize deviceToken is missing in `_Installation` and push will plainly be ignored by parse-server

**Comment by NicksonYap** [comment]
related to: https://github.com/taivo/parse-push-plugin/issues/137

**Comment by NareshMedipally** [comment]
I  have encountered the same issue. Not sure if it is an ideal fix. Was able to fix by adding another plugin “phonegap-plugin-push” to get the device token and then updated the installation class with rest api. 

https://github.com/phonegap/phonegap-plugin-push
https://docs.parseplatform.org/rest/guide/#installations-api

However you need to comment records generated by Phonegap-push  plugin in AndroidManifest.xml else you would see the two notifications on the device when you receive a push.
